### PR TITLE
Missing Chinese Translations for ELI5 prompt

### PR DESCRIPTION
**Pull request by Aliiiu**
# Missing Chinese Translations for ELI5 prompt

This is PR works on resolving missing translations for eli5

## Linked issues

closes: https://github.com/EveripediaNetwork/issues/issues/2953

### CODE REVIEW

**Comment by Adebesin-Cell** [comment]
Can you attach screenshots of before and after? I mean if there's a visual representation

**Review by Adebesin-Cell — APPROVED** [review]

**Comment by Tolu-Mals** [comment]
translations are fine, but the widget itself doesn't work though

**Comment by Aliiiu** [comment]
> translations are fine, but the widget itself doesn't work though

I think sid is working on that.
